### DISCUSSION TITLE
posts/028-paint-in-css/

### DISCUSSION THREAD

**Discussion by giscus** [Yamabuki-bakery/tasty-discussion#15] [Announcements]
# posts/028-paint-in-css/

從插圖到 CSS，手搓一切！！！

**Comment by PermeationLoop** [comment]
在黑色紙上的白色線稿看上去就沒有普通白紙上的那麼自然，，，
因爲顏色反轉疑似會違反明暗關係🤐

**Comment by PermeationLoop** [comment]
把小太陽帶回家纔是真正的幸福，，，😋

<img width="1280" height="720" alt="小太陽徽章" src="https://gist.github.com/user-attachments/assets/1619687b-8bca-481f-9bc2-04d1bd1fcfa5" />

**Comment by PermeationLoop** [comment]
## 隆重推出看板娘 v7 👒

喜歡您來 🥰

此文發出已經一年半，我仍然在折騰魔法 CSS 和插畫背景，值得讚賞（大噓）。
經過一個星期的高強度作畫，在 Krita 中砸進了快 20 小時之後，俺終於把她憋了出來 🥰

<img src="https://github.com/user-attachments/assets/5df568bc-e888-4623-8128-ae7b61e2b6a1" width="1600" height="884" alt="Time Spent on Krita" />

這張立繪有兩套配色方案，分別用於網頁的亮色和暗色模式。雙倍的工作時間，0.5 倍的預期壽命 😅，爲了 blogging 鞠躬盡瘁。然後，俺在 CSS 中使用了 opacity transition 來平滑切換兩張立繪，這個方法一開始非常穩定，🙂，直到 Bitwarden 的插件入侵了瀏覽器。這個插件會干擾網頁動畫，導致出現莫名其妙的閃爍和卡頓。Bitwarden！！！出來捱打！！！💢

> 億萬人在抱怨 Bitwarden 拖慢瀏覽器 [👉🏻Issue 1](https://github.com/bitwarden/clients/issues/14015) [👉🏻Issue 2](https://github.com/bitwarden/clients/issues/16677) [👉🏻Issue 3](https://github.com/bitwarden/clients/issues/17090)

這張畫並不完美，還有很多要改善之處，但是限於繪畫水平，我決定不再繼續死磕，而是直接自信地 deploy！！在繪製了多張 blog 插畫之後，我開始相信給 blog 畫插畫能提供一種非常好的動力去學習繪畫，畢竟有目標纔有方向。

最後祝找到這裏的幸運群友，happy blogging！

**Reply by soildCoblat** [reply]
```
導致出現莫名其妙的閃爍和卡頓
```
我用Mac裡的Chrome試了一下，確實會閃爍，我以為是插件的原因。
不過我打開Chrome的無痕瀏覽模式之後，還是會閃爍，反覆切換亮色和暗色模式之後亂動鼠標就會觸發，げんいんふめいです。

**Reply by PermeationLoop** [reply]
妳把弱小可憐的瀏覽器逼到了懸崖邊上

**Reply by PermeationLoop** [reply]
最近俺上了許多 Krenz 的真人教學，學會了許多光影二分的技法。現在想如果運用了這些技巧，俺也許能把作畫時間縮短 10 個小時（？
